src/main/kotlin/uk/gov/dluhc/emsintegrationapi/mapper/PostalVoteMapper.kt

```diff
@@ -108,13 +110,36 @@ class PostalVoteMapper(private val instantMapper: InstantMapper) {
             RejectedReasons(
                 englishReason = RejectedReason(
                     notes = englishRejectionNotes,
-                    reasons = englishRejectionReasons?.toList()
+                    reasons = englishRejectedReasonItems?.toList()?.map { it.type.value },
```

Comment:
From the postal and proxy reviews, it looks like the old `reasons` field was sending through what's now being set to `electorReason`, so we should probably be using `electorReason` here rather than `type` to maintain backwards compatibility (sorry for getting that wrong in the original comment)